samples/msal-react-sample/src/App.tsx

```diff
@@ -33,60 +59,16 @@ function App() {
                     <li><Link to="/get-access-token">Get Access Token</Link></li>
                 </ul>
                 <div className="nav-buttons">
-                    <MsalConsumer>
-                        {msal => (
-                            <AuthenticatedComponent
-                                unauthenticatedComponent={(
-                                    <button
-                                        onClick={e => {
-                                            e.preventDefault();
-                                            msal?.loginPopup({
-                                                scopes: [
-                                                    "user.read"
-                                                ]
-                                            });
-                                        }}
-                                    >
-                                        Login
-                                    </button>
-                                )}
-                            >
-                                <button
-                                    onClick={e => {
-                                        e.preventDefault();
-                                        msal?.logout();
-                                    }}
-                                >
-                                    Logout
-                                </button>
-                            </AuthenticatedComponent>
-                        )}
-                    </MsalConsumer>
+                    <AuthenticatedTemplate>
```

Comment:
Why `Template` versus `Component`?

Reply:
There may be a better alternative than `Template`, but `Component` feels redundant and might imply there is more logic behind this when it's really just mounting a defined `ReactNode` if the authentication state is correct. I was hoping `Template` would elude to it being pretty logic-less